added a feature of quick create from the input box itself..

not-meet opened the pull request:
## Issue(s)
Closes #22

##This PR introduces a new feature that allows users to create quick replies directly from the message input box using the /quick create command, eliminating the need for an input modal.

## Acceptance Criteria fulfillment
- [x] Introduce support for the /quick create command format.
- [x] Validate inputs directly within the command execution handler.
- [x]  Notify users about the success or failure of reply creation via in-app notifications.

## Proposed changes (including videos or screenshots)

- Users can now create quick replies with the following format:
/quick create <replyName> <replyBody>
- Ensures the command includes at least a name (replyName) and a body (replyBody).
- Displays an error message if the inputs are invalid or missing.
- New File for Quick Reply Logic: Contains the logic for handling /quick create.
- Updates to CommandUtility.ts: Added logic to check the input arguments. Redirects the call to the appropriate handler based on the input.

## Demo 
[Screencast from 2024-11-22 10-32-50.webm](https://github.com/user-attachments/assets/5f1572d5-6387-4951-9028-bc39269df8b6)
![Screenshot from 2024-11-22 10-38-23](https://github.com/user-attachments/assets/60997274-4b20-44c8-90e6-19a37cf2bc70)


##NOTE - the plethora of changes which are been seen in the pr are generally made because of es linting... ive checked the code and logic so from my side its all fine...
up for any changes and learning :)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/RocketChat/Apps.QuickReplies?pullRequest=23) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/RocketChat/Apps.QuickReplies?pullRequest=23) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/RocketChat/Apps.QuickReplies?pullRequest=23) it.</sub>

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -2,109 +2,140 @@ import { IUser } from '@rocket.chat/apps-engine/definition/users';
-	IModify,
-	IPersistence,
-	IRead,
+    IHttp,
```
remove these spaces

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -2,109 +2,140 @@ import { IUser } from '@rocket.chat/apps-engine/definition/users';
-	public persis: IPersistence;
-	public triggerId?: string;
-	public threadId?: string;
+    public app: QuickRepliesApp;
```
remove

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -2,109 +2,140 @@ import { IUser } from '@rocket.chat/apps-engine/definition/users';
+                const subCommand = this.params[0].toLowerCase();
+
+                if (
+                    subCommand === CommandParam.CREATE ||
```
why are we creating team in all the command ? explain me this

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -2,109 +2,140 @@ import { IUser } from '@rocket.chat/apps-engine/definition/users';
+                await this.handleSingleParam(handler);
+                break;
+            }
+            default: {
```
this should not be in default

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -2,109 +2,140 @@ import { IUser } from '@rocket.chat/apps-engine/definition/users';
+            }
+        }
+    }
+    private async handleQuickCreate(handler: Handler, args: string[]): Promise<void> {
```
we already handled default cases

VipinDevelops commented on src/modal/quickCommands/createquick.ts:
```diff
@@ -0,0 +1,72 @@
+    );
+
+    if (args.length < 3) {
+        await sendUsageMessage(modify, room, sender);
```
what is this ?

VipinDevelops commented on src/handlers/Handler.ts:
```diff
@@ -14,196 +14,213 @@ import { listReplyContextualBar } from '../modal/listContextualBar';
-			return;
-		}
-	}
+    public app: QuickRepliesApp;
```
too much bloated code remove all these spaces

VipinDevelops commented on src/modal/quickCommands/createquick.ts:
```diff
@@ -0,0 +1,72 @@
+    room: IRoom,
+    user: IUser,
+): Promise<void> {
+    const text = '/quick-create <replyName> "<replyBody>"';
```
why we have this code ?

VipinDevelops reviewed (changes requested):
Make requested changes

not-meet commented:
Hi @VipinDevelops so i reflected the changes you asked for and made them as per the requirements

**Note** 
the reason of adding the handlequickcreate function in the default is because of the number of arguments and the authentication of the keyword used as the switch case works on the parameters of length so as there can be a high chances of having uncertainty of inputs ive added quick create function in the default as it gets only called when these is a create keyword in the first index! :)

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -77,7 +77,13 @@ export class CommandUtility implements ICommandUtility {
 			}
 			default: {
-				await handler.sendDefault();
+				const subCommand = this.params[0].toLowerCase();
```
you don't have to put this in switch case you can handle this in global

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -107,4 +113,23 @@ export class CommandUtility implements ICommandUtility {
 		}
 	}
+
+	private async handleQuickCreate(handler: Handler, args: string[]): Promise<void> {
```
why dont use the function we already have for creating new replies ?

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -107,4 +113,23 @@ export class CommandUtility implements ICommandUtility {
+		  return;
+		}
+	
+		switch (args[0].toLowerCase()) {
```
why switch  case in a function that is made to create a reply ?

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -107,4 +113,23 @@ export class CommandUtility implements ICommandUtility {
+			await handler.CreateQuickReply(args);
+			break;
+		  }
+		  default: {
```
this is the third place where you please .sendDefault() is there  any other way to achieve check without repeating this

VipinDevelops commented on src/handlers/Handler.ts:
```diff
@@ -206,4 +207,20 @@ export class Handler implements IHandler {
 			return;
 		}
 	}
+	public async CreateQuickReply(args: string[]): Promise<void> {
```
we do have a method already for this

VipinDevelops reviewed (commented):
There is still a lot of room for improvement basically you are repeating a lot of code and not following best practice.

not-meet commented:
hey @VipinDevelops made changes as you asked..! now i believe the code looks much cleaner to be merged...
up if there are any other changes to be made : )

VipinDevelops commented:
Can you explain a little about changes you have made and use the code we have already for creating replies I have seen you writing some code again that already exist Either reuse the function  we have or make something new that can be use in both old and new code.

not-meet commented:
hey!! @VipinDevelops 
so I've added conditional logic in the CreateModal to take the reply name and body from CLI arguments as the original modal uses the input from the input boxes in the modal.

> I wrote new logic instead of modifying the existing modal to avoid conflicts. While I understand the concern about calling the handler in the default switch case instead of passing it globally, doing so avoids issues where the global call still triggers the switch case unintentionally.

VipinDevelops commented on README.md:
```diff
@@ -76,6 +76,13 @@ By selecting quick replies instead of typing manually, agents/users can respond
 -   **`/quick ai`**: Use AI to generate replies
 -   **`/quick help`**: Get help with Quick Reply
 -   **`/qs <reply name>`**: Quickly search and send a reply by name
+-   **`/quick create <name> <message>`**:  Create a quick reply directly from the input box with a name and message
```
add screen shot how it look

not-meet replied:
![Screenshot from 2025-02-23 02-29-31](https://github.com/user-attachments/assets/a6e6c41b-0b2b-4245-ae21-fc4c7912671b)

VipinDevelops replied:
put create command together , explain how double quotes works to name a reply

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -65,8 +65,17 @@ export class CommandUtility implements ICommandUtility {
 			triggerId: this.triggerId,
 			threadId: this.threadId,
 			language,
+			args: this.params,
```
?

not-meet replied:
so basically the handler needs args to be passed to get the name  and body from the cli so thats why i added params here to pass it down to handler function

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -76,7 +85,7 @@ export class CommandUtility implements ICommandUtility {
 				break;
 			}
-			default: {
+			default: {				
```
?

VipinDevelops commented on src/definition/handlers/IHandler.ts:
```diff
@@ -10,4 +10,5 @@ export interface IHandler extends Omit<ICommandUtilityParams, 'params'> {
 
 export type IHanderParams = Omit<ICommandUtilityParams, 'params'> & {
 	language: Language;
+	args? : string[];
```
?

VipinDevelops commented on src/handlers/Handler.ts:
```diff
@@ -64,13 +67,19 @@ export class Handler implements IHandler {
 			this.modify,
 			this.room,
 			this.language,
+			this.args ?? [],
```
?

VipinDevelops commented on src/handlers/Handler.ts:
```diff
@@ -64,13 +67,19 @@ export class Handler implements IHandler {
 			return;
 		}
 
+		if (!modal) {
```
?

VipinDevelops commented on src/modal/createModal.ts:
```diff
@@ -25,7 +27,29 @@ export async function CreateReplyModal(
 	language: Language,
-): Promise<IUIKitSurfaceViewParam | Error> {
+	args: string[],
+): Promise<IUIKitSurfaceViewParam | Error | void> {
```
no change here

VipinDevelops commented:
Again fix PR name , please follow the standard for everything you do

VipinDevelops commented on src/commands/QuickCommand.ts:
```diff
@@ -46,7 +48,6 @@ export class QuickCommand implements ISlashCommand {
 
-		const commandUtility = new CommandUtility(commandUtilityParams);
-		await commandUtility.resolveCommand();
+		await new CommandUtility(commandUtilityParams).resolveCommand();
```
why ?

VipinDevelops commented on src/modal/createModal.ts:
```diff
@@ -56,6 +59,7 @@ export async function CreateReplyModal(
 			label: labelReplyBody,
 			optional: false,
 			multiline: true,
+			initialValue: args.length > 2 ? args.slice(2).join(' ') : undefined,
```
i dont think that's how you should do this

VipinDevelops commented on QuickRepliesApp.ts:
```diff
@@ -36,9 +36,20 @@ import { settings } from './src/config/settings';
 		super(info, logger, accessors);
 	}
+
+	public setCommandParams(userId: string, params: string[]) {
```
what is this ?

VipinDevelops commented on QuickRepliesApp.ts:
```diff
@@ -98,12 +109,15 @@ export class QuickRepliesApp extends App {
 		persistence: IPersistence,
 		modify: IModify,
 	) {
+		const userId = context.getInteractionData().user.id;
```
again what is this ?

VipinDevelops commented on QuickRepliesApp.ts:
```diff
@@ -98,12 +109,15 @@ export class QuickRepliesApp extends App {
 		modify: IModify,
 	) {
+		const userId = context.getInteractionData().user.id;
+		const storedParams = this.getCommandParams(userId);
```
why are you strong params makes no sense to me

VipinDevelops commented on README.md:
```diff
@@ -76,6 +76,13 @@ By selecting quick replies instead of typing manually, agents/users can respond
 -   **`/quick help`**: Get help with Quick Reply
 -   **`/qs <reply name>`**: Quickly search and send a reply by name
+-   **`/quick create <name> <message>`**:  Create a quick reply directly from the input box with a name and message
+
```
add SS

VipinDevelops replied:
add SS man why you are not making requested changes before asking for review.

VipinDevelops commented on src/handlers/ExecuteViewSubmitHandler.ts:
```diff
@@ -34,16 +34,17 @@ import { AIusagePreference } from '../definition/helper/userPreference';
 		protected readonly http: IHttp,
 		protected readonly persistence: IPersistence,
 		protected readonly modify: IModify,
+		protected readonly params: ICommandUtilityParams['params'],
```
??

VipinDevelops commented on src/handlers/ExecuteViewSubmitHandler.ts:
```diff
@@ -80,6 +81,7 @@ export class ExecuteViewSubmitHandler {
 						view,
 						language,
 						triggerId,
+						this.params
```
?

VipinDevelops commented on src/handlers/ExecuteViewSubmitHandler.ts:
```diff
@@ -127,9 +130,11 @@ export class ExecuteViewSubmitHandler {
-		const name = nameStateValue ? nameStateValue.trim() : '';
-		const body = bodyStateValue ? bodyStateValue.trim() : '';
+		
+		const argsName = params.length > 1 ? params[1] : undefined;
```
why undefined that should not be possible

VipinDevelops commented on src/handlers/ExecuteViewSubmitHandler.ts:
```diff
@@ -127,9 +130,11 @@ export class ExecuteViewSubmitHandler {
+		
+		const argsName = params.length > 1 ? params[1] : undefined;
+		const argsBody = params.length > 2 ? params.slice(2).join(' ') : undefined;
+		const name = argsName ? argsName : nameStateValue ? nameStateValue.trim() : '';
```
this ternary operator is doing simple thing in much complex way

VipinDevelops commented on src/handlers/Handler.ts:
```diff
@@ -47,6 +48,8 @@ export class Handler implements IHandler {
 		this.triggerId = params.triggerId;
 		this.threadId = params.threadId;
 		this.language = params.language;
+		this.args = params.args;
```
?

VipinDevelops reviewed (changes requested):
Alot of changes required

not-meet commented:
hi! sorry for the delay for now i have changed the pr name and in the recent commit was just trying this approach, I  will
go through the reviews and make a final commit as soon as possible

not-meet commented on src/handlers/ExecuteViewSubmitHandler.ts:
```diff
@@ -119,14 +119,12 @@ export class ExecuteViewSubmitHandler {
-			view.state?.[CreateModalEnum.REPLY_BODY_BLOCK_ID]?.[
-				CreateModalEnum.REPLY_BODY_ACTION_ID
-			];
+		const nameStateValue = view.state && Object.values(view.state)
```
sot he reason to add this new logic of fetching the states of name and body is because when you try to create a reply from the prefilled values the view state dont have the block id 

![Screenshot from 2025-02-23 02-22-10](https://github.com/user-attachments/assets/9f0c7c59-a242-4a63-a593-596b4293eb02)

where as the name and body provided from the modal have the block id thus to take the values of action id which is the only thing we need in both cases i added find logic which works well in both the cases as it only takes out the reply action id

VipinDevelops replied:
this code is so bad adding find don't make sense to me here and again you are not using enums , please find why enum are good and how we are using in this case.

not-meet commented on src/handlers/Handler.ts:
```diff
@@ -64,13 +69,15 @@ export class Handler implements IHandler {
 			this.modify,
 			this.room,
 			this.language,
+			cliName,
```
rather than passing the args to the create modal i have passed cliname and clibody to the modal for the initial values

not-meet commented:
hey @VipinDevelops i have made the changes as asked and also provided the reasoning's behind all the changes also updated the pr proposal 

**I have added the new demo in the proposal as well** 
i request you to kindly go through that as well

VipinDevelops commented on src/commands/CommandUtility.ts:
```diff
@@ -76,7 +85,7 @@ export class CommandUtility implements ICommandUtility {
 				break;
 			}
-			default: {
+			default: {	
```
?

VipinDevelops commented on src/handlers/Handler.ts:
```diff
@@ -56,6 +59,8 @@ export class Handler implements IHandler {
 	}
 
 	public async CreateReply(): Promise<void> {
+		const cliName = this.args?.[1] || '';
```
could we give them better naems like initial name or something ? or can we just make one object that have both values

VipinDevelops reviewed (changes requested):
read how I named my intial PR and use similar name I won't allow any PR that is not following good naming and description practicc.

VipinDevelops commented on src/handlers/Handler.ts:
```diff
@@ -56,6 +59,8 @@ export class Handler implements IHandler {
 	}
 
 	public async CreateReply(): Promise<void> {
+		const initialReplyName = this.args?.[1] || '';
```
initial values are optional why OR here ? just don't pass it if its not there

VipinDevelops commented on src/modal/common/inputElementComponent.ts:
```diff
@@ -50,6 +50,7 @@ export function inputElementComponent(
 		text: label,
 		element: plainTextInputElement,
 		optional,
+		blockId,
```
??

VipinDevelops replied:
???

VipinDevelops commented on src/modal/createModal.ts:
```diff
@@ -25,7 +25,10 @@ export async function CreateReplyModal(
 	modify: IModify,
 	room: IRoom,
 	language: Language,
+	initialReplyName: string,
```
we can make it a object but fine for now

VipinDevelops reviewed (changes requested):
you are close

VipinDevelops commented on README.md:
````diff
@@ -83,6 +83,13 @@ By selecting quick replies instead of typing manually, agents/users can respond
+
+#### Example:
+```sh
+/quick create "greeting" Hello! How have you been?
````
This is not what I ment 

How double quotes make the multiple words a single text for body use

VipinDevelops commented on src/definition/handlers/IHandler.ts:
```diff
@@ -10,4 +10,5 @@ export interface IHandler extends Omit<ICommandUtilityParams, 'params'> {
 
 export type IHanderParams = Omit<ICommandUtilityParams, 'params'> & {
 	language: Language;
+	params?: string[];
```
Why optinal?

VipinDevelops replied:
not answered ?

VipinDevelops replied:
again not answered?

VipinDevelops reviewed (changes requested):
When you make requested changes also share about what you changed in comment or reply

not-meet commented:
hey @VipinDevelops i've added the comments with detailed explanation of what changes i have made

not-meet commented:
Ive made the required changes in the readme file 
![Screenshot from 2025-02-27 06-14-51](https://github.com/user-attachments/assets/23bba5ef-e011-4510-b768-e3bc61c9e3ea)

added the detailed explanation and examples how "quotes" work . 

**NOTE**-i have also added comments in all the changes i have made with detailed reasoning's kindly go through them as well

VipinDevelops commented on README.md:
````diff
@@ -86,12 +86,38 @@ By selecting quick replies instead of typing manually, agents/users can respond
 -   **`/qs <reply name>`**: Quickly search and send a reply by name
 
+
+#### Creating Quick Replies from Message Box
+**`/quick create "<name>" <message>`**: Create a quick reply using the message box
+
+When you use this command:
+1. A modal window opens automatically
+2. The name and replybody field is prefilled with your specified values
+3. Simply review and click the Submit button to save your quick reply
+
+The command works as follows:
+- The name must be enclosed in quotes `"..."` if it contains multiple words
+- Everything after the name is treated as the message body
+- Without quotes, only the first word would be taken as the name
+
+#### Examples:
+```sh
+# Single word name (quotes optional)
+/quick create greeting Hello! How have you been?
+
+# Multi-word name (quotes required)
+/quick create "welcome message" Welcome to our channel! How can I help you?
+```
+
+Both examples will open a modal with the name and message already filled in. Just click Submit to save your quick reply, which can then be used with **`/qs <name>`**.
+
````
That is just to much for such simple thing 

 and can't we just do like 

/quick create "you can use double quotes to write the name with spaces" you write everything else as a body ??

VipinDevelops replied:
this just looks like AI-generated markdown to me

VipinDevelops commented on src/handlers/Handler.ts:
```diff
@@ -35,6 +35,7 @@ export class Handler implements IHandler {
 	public triggerId?: string;
 	public threadId?: string;
 	public language: Language;
+	public params?: string[];
```
stilll ?

VipinDevelops replied:
man ?

VipinDevelops reviewed (changes requested):
requested more changes 
 don't tag me before you reply to all the old messages

not-meet commented:
hey @VipinDevelops 
i have made the requested changes in the readme 
![Screenshot from 2025-02-28 04-45-22](https://github.com/user-attachments/assets/1cbcba79-6213-4da1-be86-fd345baca312)

i have added examples as well

not-meet commented:
I have also replied to the asked questions in the messages in the review section in each file still to ease it out

### why i added params as optional? 

- so the reason to make this optional is because not all the handler implementations such as executeActionbutton or executeBlockAction, they dont need params to run thus i made this optional as it removes unnecesary addition of parameter

### why i added blockId in the input element component? 

-

not-meet commented:
@VipinDevelops i was thinking about making a new pr for the same as this pr now has too many conversation messages and commits thus makes it a bit more clutterd and not easy to get through all the messages and conversation

**i am thinking if you allow me to make a new pr for this issue with the same commits and solution**

VipinDevelops commented:
> hey @VipinDevelops i have made the requested changes in the readme ![Screenshot from 2025-02-28 04-45-22](https://private-user-images.githubusercontent.com/140915491/417816395-1cbcba79-6213-4da1-be86-fd345baca312.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3MjExNDksIm5iZiI6MTc0MDcyMDg0OSwicGF0aCI6Ii8xNDA5MTU0OTEvNDE3ODE2Mzk1LTFjYmNiYTc5LTYyMTMtNGRhMS1iZTg2LWZkMzQ1YmFjYTMxMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIyOFQwNTM0MDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMjBiMTI3NzBkOTBjNjYxNzVlYzg2NGMyODc1OTk0ZGVmNDBmNjg4NzBiOTNiYTljNTZmMTZhNGU4YzI2MmE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.3JgxpePW3grXsgtzaAE-mnTj4RYVxdGgR2PWUn45PEY)
> 
> i have added examples as well

Super bad example Can't understand anything make it simple and short

VipinDevelops commented:
> I have also replied to the asked questions in the messages in the review section in each file still to ease it out
> 
> ### why i added params as optional?
> * so the reason to make this optional is because not all the handler implementations such as executeActionbutton or executeBlockAction, they dont need params to run thus i made this optional as it removes unnecesary addition of parameter

Just remove it if not needed why make it optional ?

VipinDevelops commented:
> @VipinDevelops i was thinking about making a new pr for the same as this pr now has too many conversation messages and commits thus makes it a bit more clutterd and not easy to get through all the messages and conversation
> 
> **i am thinking if you allow me to make a new pr for this issue with the same commits and solution**

NO we won't make new PR's because of  TOO MANY conversations

not-meet commented:
> > I have also replied to the asked questions in the messages in the review section in each file still to ease it out
> > ### why i added params as optional?
> > 
> > * so the reason to make this optional is because not all the handler implementations such as executeActionbutton or executeBlockAction, they dont need params to run thus i made this optional as it removes unnecesary addition of parameter
> 
> Just remove it if not needed why make it optional ?

See Handler function is implemented in three files executeBlockActionhandler , executeActionButtonHandler and commandUtilty where only the command utility needs params as the createReply uses params to get the initial values 

![Screenshot from 2025-02-28 20-13-34](https://github.com/user-attachments/assets/1a397e53-604e-40f7-9ea0-8d189976566d)
![Screenshot from 2025-02-28 20-13-47](https://github.com/user-attachments/assets/bc359836-1525-4ca6-9094-5cf89712c169)
![Screenshot from 2025-02-28 20-13-51](https://github.com/user-attachments/assets/bdb3da45-1e69-46e6-9121-5c616613c7b6)


here are all the places where the handler is used 
**Notice how only one handler needs params** - thus i added params as optional as it can only be passed where it is necessary just like how threadID args works

not-meet commented:
> > hey @VipinDevelops i have made the requested changes in the readme ![Screenshot from 2025-02-28 04-45-22](https://private-user-images.githubusercontent.com/140915491/417816395-1cbcba79-6213-4da1-be86-fd345baca312.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA3MjExNDksIm5iZiI6MTc0MDcyMDg0OSwicGF0aCI6Ii8xNDA5MTU0OTEvNDE3ODE2Mzk1LTFjYmNiYTc5LTYyMTMtNGRhMS1iZTg2LWZkMzQ1YmFjYTMxMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIyOFQwNTM0MDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMjBiMTI3NzBkOTBjNjYxNzVlYzg2NGMyODc1OTk0ZGVmNDBmNjg4NzBiOTNiYTljNTZmMTZhNGU4YzI2MmE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.3JgxpePW3grXsgtzaAE-mnTj4RYVxdGgR2PWUn45PEY)
> > i have added examples as well
> 
> Super bad example Can't understand anything make it simple and short

![Uploading Screenshot from 2025-02-28 21-08-27.png…]()

How about this i have changed the examples and made them quite more clear

VipinDevelops commented on README.md:
```diff
@@ -86,14 +86,26 @@ By selecting quick replies instead of typing manually, agents/users can respond
+**`/quick create "<name>" <message>`**: "if your name has multiple words with space, use double quotes" and write everything else as a body!
+
+Example: `/quick create "schedule meeting" let's have a quick meet in an hour!`
+
```
This is not what I meant.

Can you please avoid writing a lot in the README to explain really simple things?

VipinDevelops reviewed (changes requested):
There are still a lot of changes needed for this.

We are just making it longer—try to fix it this time
